cdk-infra/package.json

```diff
@@ -9,12 +9,13 @@
     "watch": "tsc -w",
     "test": "jest",
     "cdk": "cdk",
-    "deploy:enhanced": "cdk deploy -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --all",
-    "deploy:enhanced:worker": "cdk deploy auto-builder-stack-enhancedApp-worker -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:enhanced:webhooks": "cdk deploy auto-builder-stack-enhancedApp-webhooks -c enhanced=true -c customFeatureName=enhancedApp -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:feature": "cdk deploy -c enhanced=true -c env=dotcomstg --require-approval never --all",
-    "deploy:feature:worker": "cdk deploy  -c enhanced=true -c env=dotcomstg --require-approval never --exclusively",
-    "deploy:feature:webhooks": "cdk deploy -c enhanced=true -c env=dotcomstg --require-approval never --exclusively"
+    "deploy:enhanced": "cdk deploy -c enhanced=true -c customFeatureName=enhancedApp -c env=prd --require-approval never --all",
```

Comment:
Are these commands expected to be tied to the `prd` environment by default?

Reply:
yep! Would you like me to specify the environment in the command name? I could do something like `"deploy:enhanced:prd"` or something along those lines.

Reply:
Hm, `deploy:enhanced:prd` might work. I was just concerned about the GitHub Actions for staging calling these commands for the `prd` environment